setup.php

```diff
@@ -28,7 +28,7 @@
  * -------------------------------------------------------------------------
  */
 
-define('PLUGIN_MREPORTING_VERSION', '1.8.6');
+define('PLUGIN_MREPORTING_VERSION', '1.8.7');
```

Comment:
You don't need to change the plugin version, this is handled by the github action.